### DISCUSSION TITLE
Problem to Initializing properties dynamically, | Livewire 3

### DISCUSSION THREAD

**Discussion by gctom** [livewire/livewire#8549] [1. Help]
I have list from database

$items = Item::all();

foreach($items as $item)

for this $item 

I need to initialize this as a public $item1 item2 etc.

because.

I need to display list item on blade.

while wire:model="{{$item}} 
that need to point to public property or bind to it.

form this list item I will iterate multiple select options

<select wire:model="{{$item}} >
options
</select>
<select wire:model="{{$item}} >
options
</select>
<select wire:model="{{$item}} >
options
</select>

I can't initialize hard code if the user adds any select attributes from admin then it will not come on the livewire component class, only display on blade.

I tried multiple methods by ChatGPT but not working.

**Comment by silvershades** [comment]
```
foreach($items as $itemKey =>$item){
    <select wire:model="$items{$itemKey}"> 
    
    </select>
}
```
